.changeset/smart-panthers-collect.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/theme-check-common': patch
+---
+
+invisible_if should not warn on use of 'section' in a block file
```

Comment:
Not sure what this file is but this should be `visible_if` as opposed to `invisible_if`

```suggestion
visible_if should not warn on use of 'section' in a block file
```